astropy/coordinates/baseframe.py

```diff
@@ -691,10 +695,14 @@ def isscalar(self):
         return self.has_data and self.data.isscalar
 
     @classmethod
-    def get_frame_attr_names(cls):
+    def get_frame_attr_defaults(cls):
         return {name: getattr(cls, name)
                 for name in cls.frame_attributes}
 
+    @classmethod
+    def get_frame_attr_names(cls):
+        return list(cls.frame_attributes)
```

Comment:
It would be better if these methods had docstrings, but the because there weren't any before either then this shouldn't necessarily prevent merging.